tests/unittests/Accelerate/vImageTest.mm

```diff
@@ -821,7 +821,7 @@ static void vImageTestSetAlphaAndUnpremultiply(CGImageRef imageRef, vImage_Buffe
     vImageTestBufferFree(&rgbDest);
 }
 
-TEST(Accelerate, AlphaUnpremultiply) {
+DISABLED_TEST(Accelerate, AlphaUnpremultiply) {
```

Comment:
please see the other comments.

---
In reply to: [85251339](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r85251339) [](ancestors = 85251339)